plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -59,6 +59,8 @@
                 <div class="pk-emptyDataTable">
                 {% if showReportDataWasPurgedMessage is defined and showReportDataWasPurgedMessage %}
                     {{ 'CoreHome_DataForThisReportHasBeenPurged'|translate(deleteReportsOlderThan) }}
+                {% elseif showPluginArchiveDisabled %}
```

Comment:
Might be better to check if the variable is defined, otherwise it might throw an exception if that is not the case.
```suggestion
                {% elseif showPluginArchiveDisabled is defined and showPluginArchiveDisabled %}
```